docs/99-pure-storage-attach.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-pure-storage-attach
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - path: /etc/multipath.conf
+        mode: 384
+        filesystem: root
+        contents:
+          source: data:,devices%20%7b%0a%20%20device%20%7b%0a%20%20%20%20%20%20%20%20vendor%20%22PURE%22%0a%20%20%20%20%20%20%20%20product%20%22FlashArray%22%0a%20%20%20%20%20%20%20%20fast_io_fail_tmo%2010%0a%20%20%20%20%20%20%20%20path_grouping_policy%20%22group_by_prio%22%0a%20%20%20%20%20%20%20%20failback%20%22immediate%22%0a%20%20%20%20%20%20%20%20prio%20%22alua%22%0a%20%20%20%20%20%20%20%20hardware_handler%20%221%20alua%22%0a%20%20%20%20%20%20%20%20max_sectors_kb%204096%0a%20%20%20%20%7d%0a%7d
```

Comment:
Do we want to hard-code the `multipath.conf` setting here?
We have a script that they can manually run on each worker nodes: https://github.com/dsupure/multipath_tool/tree/master/scripts/multipath-utils
Users can customized their settings.